lib/dul_arclight/traject/ead2_config.rb

```diff
@@ -301,6 +314,18 @@
   accumulator.replace range.years
 end
 
+to_field 'date_range_sim', extract_xpath('/ead/archdesc/did/unittitle/unitdate') do |_record, accumulator, context|
+  unless context.output_hash['date_range_sim']
+    # logger.debug accumulator
+    clean_range = accumulator.map { |v| v.gsub(/([^()]*)\(.*\)/, '\1').tr('^0-9\\-/ ', '').strip }
+                      .select { |date| date.match? /^\d\d\d\d[-\/]\d\d\d\d$/}
+    # logger.debug clean_range
+    range = Arclight::YearRange.new(clean_range.include?('/') ? clean_range : clean_range.map { |v| v.tr('-', '/') })
+    accumulator.replace range.years
+  end
+end
```

Comment:
I'm finding this very confusing. I think what it's saying is:
* Throw out everything between an open-paren and a close-paren
  * "1111" => "1111"
  * "1111 (abcd)" => "1111 "
  * "1111 (abcd) 2222 (efgh)" => "1111" (greedy between the parens)
  * "(abcd)" => ""
  * ")1111" => ")1111"
  * "1)1111 (abcd)" => "1)1111 (abcd)" (right-paren screws things up)
  * "(abcd) 1111 (efgh)" => ""
* Throw out everything that's not a digit, dash, slash, or space
* Strip off any spaces (which, in fact, the `.tr` has already done)
* Accept things that match `%r(\A\d{4}[-/]\d{4}\Z)`

So, `clean_range.include?('/')` is _never_ going to be true, since it's just looking for a string that's a single '/', and you've already ditched everything not of the form 1111[-/]1111

I see that `Arclight::YearRange` will take a list of strings of the form `1111/2222` but I'm not sure what types of data you're trying to clean up with this.